hack/testing/logging.sh

```diff
@@ -76,6 +76,58 @@ function cleanup()
     exit $out
 }
 
+function wait_for_latest_build_complete() {
+
+  interval=30
+  waittime=120
+
+  local bc=$1
+  local lastVersion=$(oc get bc $bc -o jsonpath='{.status.lastVersion}')
+  local status
+
+  for (( i = 1; i <= $waittime; i++ )); do
+    status=$(oc get build -o jsonpath='{.items[?(@.metadata.name=="'$bc'-'$lastVersion'")].status.phase}')
```

Comment:
I think this would be clearer/simpler as:

    status=$(oc get build/$bc-$lastVersion -o jsonpath='{.status.phase}')